src/general/OptionsMenu.tscn

```diff
@@ -825,6 +826,13 @@ margin_right = 511.0
 margin_bottom = 27.0
 rect_min_size = Vector2( 300, 0 )
 
+[node name="ScreenShot folder" type="Button" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer"]
+margin_top = 349.0
+margin_right = 511.0
+margin_bottom = 379.0
+action_mode = 0
+text = "Screenshot Folder"
```

Comment:
I think the button should say "Open Screenshot Folder"

Reply:
ok I changed that